ghastly/core.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+
+
+class Core:
+    '''
+    Parent class for Core objects.  Because the Core class doesn't define
+    a specific geometry, the Core class should not be used directly.
+    '''
+
+    def __init__(self, x_c, y_c, z_max, z_min):
+        '''
+        Initializes a single instance of a Core object.
+
+        Parameters
+        ----------
+        x_c : float
+            Coordinate of the cone's center on the x_axis.
+        y_c : float
+            Coordinate of the cone's center on the y_axis.
+        z_max : float
+            Z-coordinate of the cylinder's top.
+        z_min : float
+            Z-coordinate of the cylinder's bottom.
```

Comment:
Doesn't define geometry but this specifies cone center and cylinder height?

Reply:
That's a typo!  Thanks for catching it.  it should have been changed to "core element".